spk/couchpotatoserver/src/service-setup.sh

```diff
@@ -0,0 +1,22 @@
+PYTHON_DIR="/usr/local/python"
+GIT_DIR="/usr/local/git"
+PATH="${SYNOPKG_PKGDEST}/bin:${SYNOPKG_PKGDEST}/env/bin:${PYTHON_DIR}/bin:${GIT_DIR}/bin:${PATH}"
+PYTHON="${SYNOPKG_PKGDEST}/env/bin/python"
+VIRTUALENV="${PYTHON_DIR}/bin/virtualenv"
+COUCHPOTATOSERVER="${SYNOPKG_PKGDEST}/share/CouchPotatoServer/CouchPotato.py"
+CFG_FILE="${SYNOPKG_PKGDEST}/var/settings.conf"
+
+GROUP="sc-media"
+
+SERVICE_COMMAND="${PYTHON} ${COUCHPOTATOSERVER} --daemon --pid_file ${PID_FILE} --config_file ${CFG_FILE}"
+
+service_postinst ()
+{
+    # Create a Python virtualenv
+    ${VIRTUALENV} --system-site-packages ${SYNOPKG_PKGDEST}/env >> ${INST_LOG} 2>&1
```

Comment:
OK for Python environment, but am I wrong or `pip install` from `requirements.txt` file is required and lacking ?

Reply:
Couchpotato has no requirements, everything comes included :)

Reply:
OK so in that specific case, is a pip environment really required ??

Reply:
Not really, but it's considered best practice to make sure one package can't mess with anything. And doesn't take a lot of space so seems like a good idea always.